airflow/jobs/scheduler_job_runner.py

```diff
@@ -1391,6 +1392,7 @@ def _create_dag_runs_asset_triggered(
                         run_type=DagRunType.ASSET_TRIGGERED,
                         logical_date=logical_date,
                         data_interval=data_interval,
+                        run_after=max(logical_dates.values()),
```

Comment:
There’s a task to make asset-triggered runs have None logical_date instead, so we’ll need to change this again soon. This is good enough for now.